### PR TITLE
Add missing translation

### PR DESCRIPTION
**Pull request by pin3da**
Add missing translation #1608

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by daxlab — APPROVED** [review]

File: locale/es/download/current.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ downloads:
     intro: >
         Descargue el código fuente de Node.js o un instalador pre-compilado para su plataforma, y comience a desarrollar hoy.
     currentVersion: Versión actual
-    buildInstructions: Building Node.js from source on supported platforms
+    buildInstructions: Compilando Node.js desde el código fuente en las plataformas soportadas
```
Comment (WaleedAshraf) [inline locale/es/download/current.md:15]: 👍 for fixing this. :)
I don't know Spanish,
But Google translator (which is not that accurate) is translating this Spanish into English as:
> Compiling Node.js from source code on supported platforms.

And when I do original English -> Spanish
> Construyendo Node.js desde la fuente en las plataformas compatibles.

Comment (WaleedAshraf) [reply locale/es/download/current.md]: 👍 for fixing this. :)

Comment (pin3da) [reply locale/es/download/current.md:15]: I added the translation suggested in #1608. I can make any change if needed :+1:

Comment (ghost) [reply locale/es/download/current.md:15]: I think it's fine as it is, native speakers already agreed about this translation in #1608

**Review by WaleedAshraf — APPROVED** [review]
LGTM. 👍👍

**Pull request merged**